arch/posix/core/posix_core.c

```diff
@@ -54,7 +54,7 @@
 #define ERPREFIX   PREFIX"error on "
 #define NO_MEM_ERR PREFIX"Can't allocate memory\n"
 
-#if POSIX_ARCH_DEBUG_PRINTS
+#if defined(POSIX_ARCH_DEBUG_PRINTS) && (POSIX_ARCH_DEBUG_PRINTS != 0)
```

Comment:
This macro (and the same change below) is defined 20 lines ahead in this same file. If you really want to change it like this go head, but it just seems quite funny code..

Reply:
yeah, I'll remove this. But the original code itself is weird, this value is hardcoded, it seems something used for development